opcua.cpp

```diff
@@ -201,10 +167,14 @@ static bool IsValidParentReferenceId(char *referenceId)
  */
 void OPCUA::dataChange(const char *nodeId, const SOPC_DataValue *value)
 {
+	if (m_stopped.load() || !m_readyForData.load())
```

Comment:
This is a bit of a semantic change, but I know why you are doing it. I would love to know if we could reinstate the previous behaviour when subscribing to lots of nodes.

Reply:
I believe we could go back to the original. We were trying to tackle the lack of responsiveness while loading data. I was also puzzled when the shutdown command never appeared. Now that the rest of the code has settled, I could try these use cases again.